views/partials/header.njk

```diff
@@ -1,38 +1,54 @@
+                <a href="/jobs" class="text-gray-700 hover:text-blue-600 font-medium px-3 py-2 transition-colors duration-200">Jobs</a>
+                <a href="#" class="text-gray-700 hover:text-blue-600 font-medium px-3 py-2 transition-colors duration-200">About</a>
+                <a href="#" class="text-gray-700 hover:text-blue-600 font-medium px-3 py-2 transition-colors duration-200">Contact</a>
+            </nav>
+
+            <!-- Desktop CTA Button -->
+            <div class="hidden md:flex flex-shrink-0 ml-4">
+                <button class="bg-blue-600 hover:bg-blue-700 text-white px-4 py-2 rounded-md font-medium transition-colors duration-200">
+                    Get Started
+                </button>
+            </div>
+
             <!-- Mobile menu button -->
-            <div class="lg:hidden">
-                <div class="dropdown">
-                    <div tabindex="0" role="button" class="btn btn-ghost">
+            <div class="md:hidden ml-auto">
+                <div class="dropdown dropdown-end">
+                    <div tabindex="0" role="button" class="btn btn-ghost btn-square">
                         <i data-lucide="menu" class="h-5 w-5"></i>
                     </div>
-                    <ul tabindex="0"
-                        class="menu menu-sm dropdown-content mt-3 z-[1] p-2 shadow bg-white rounded-box w-52">
-                        <li><a href="/" class="text-gray-700 hover:text-blue-600">Home</a></li>
-                        <li><a href="/jobs" class="text-gray-700 hover:text-blue-600">Jobs</a></li>
-                        <li><a href="#" class="text-gray-700 hover:text-blue-600">About</a></li>
-                        <li><a href="#" class="text-gray-700 hover:text-blue-600">Contact</a></li>
+                    <ul tabindex="0" class="menu menu-sm dropdown-content mt-3 z-[1] p-2 shadow-lg bg-white rounded-lg w-56 border border-gray-200">
+                        <li><a href="/" class="text-gray-700 hover:text-blue-600 hover:bg-blue-50 px-4 py-3 rounded-md transition-colors duration-200">
+                            <span>Home</span>
+                        </a></li>
+                        <li><a href="/jobs" class="text-gray-700 hover:text-blue-600 hover:bg-blue-50 px-4 py-3 rounded-md transition-colors duration-200">
+                            <span>Jobs</span>
+                        </a></li>
+                        <li><a href="#" class="text-gray-700 hover:text-blue-600 hover:bg-blue-50 px-4 py-3 rounded-md transition-colors duration-200">
+                            <span>About</span>
+                        </a></li>
+                        <li><a href="#" class="text-gray-700 hover:text-blue-600 hover:bg-blue-50 px-4 py-3 rounded-md transition-colors duration-200">
+                            <span>Contact</span>
+                        </a></li>
```

Comment:
that's a lot of repetition that could be extracted to a reusable component.
Nunjucks has a concept of macros, you can read it about it here:
https://mozilla.github.io/nunjucks/templating.html#macro

Happy to merge the current version though and refactor later.